test/debug/break_test.rb

```diff
@@ -206,6 +206,18 @@ def test_stops_at_correct_place_when_breakpoint_set_in_empty_line
       end
     end
 
+    def test_conditional_breakpoint_stops_for_repeated_iterations
```

Comment:
to make sure the change doesn't prevent the same breakpoint from stopping multiple times.